plugin/http/instrumentation/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package instrumentation provides OpenCensus instrumentation for the standard library http package.
+package instrumentation // import "go.opencensus.io/plugin/http/instrumentation"
```

Comment:
Can we call this package httpplugin or something specifically named after http?

We call the grpc plugin grpc, and I was wondering if we can name it to grpcopencensus. Maybe, we can do the same here.

Reply:
what about "go.opencensus.io/plugin/ochttp" "go.opencensus.io/plugin/ocgrpc"?

Reply:
For some reason, I'd prefer opencensus to be in the name to make it distinguishable. I'd expect most users to directly depend on these two packages even if they are not involved in OpenCensus any further.

I have no strong opinions though. /cc @odeke-em for more input.